### PR TITLE
From cookiecutter

### PR DESCRIPTION
**Pull request by Alex-Lewandowski**
rebuilt from cookiecutter

### CODE REVIEW

File: profile_label/package.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   "bugs": {
-    "url": "https://github.com/ASFOpenSARlab/opensarlab_doc_link_widget/issues"
+    "url": "https://github.com/ASFOpenSARlab/opensarlab-extensions/issues"
   },
   "license": "BSD-3-Clause",
   "author": {
-    "name": "ASF OpenSARlab",
-    "email": "uaf-jupyterhub-asf@alaska.edu"
+    "name": "ASF OpenSARlab Team",
+    "email": "aflewandowski@alaska.edu"
```
Comment (emlundell) [inline profile_label/package.json]: Better to use uaf-jupyterhub-asf@alaska.edu?

File: profile_label/profile_label/handlers.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import tornado
+from tornado.web import StaticFileHandler
+
+class RouteHandler(APIHandler):
+    # The following decorator should be present on all verb methods (head, get, post,
+    # patch, put, delete, options) to ensure only authorized user can request the
+    # Jupyter server
+    @tornado.web.authenticated
+    def get(self):
+        profile_name = os.environ.get('opensarlab_profile_name', 'No environment variable "opensarlab_profile_name" found!')
```
Comment (emlundell) [inline profile_label/profile_label/handlers.py]: Should this be `opensarlab_profile_name` or `osl_profile_name` or `OPENSARLAB_PROFILE_NAME` or whatever? We need to be consistent so other extensions can use the same env.

File: profile_label/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  id: 'jupyterlab-topbar-profile-label',
+  autoStart: true,
+  requires: [ITopBar],
+  activate: async (app: JupyterFrontEnd, topBar: ITopBar) => {
+    let data = null;
+    try {
+      data = await requestAPI<any>('profile-label');
+      console.log(data);
+    } catch (reason) {
+      console.error(`Error on GET /jlab-ext-example/profile-label.\n${reason}`);
```
Comment (emlundell) [inline profile_label/src/index.ts]: References to `jlab-ext-example` for the subpath need to be changed to something else. Perhaps `profile-label`?

File: profile_label/profile_label/handlers.py (hunk 2 above)
Comment (Alex-Lewandowski) [reply profile_label/profile_label/handlers.py]: updated

File: profile_label/src/index.ts (hunk 3 above)
Comment (Alex-Lewandowski) [reply profile_label/src/index.ts]: updated

**Pull request merged**